src/main/resources/theme/vault/replay/replay_card.fxml

```diff
@@ -97,8 +98,16 @@
                 <Region styleClass="icon,uninstall-icon"/>
             </graphic>
         </Button>
+        <Button fx:id="rehostButton" mnemonicParsing="false" styleClass="primary-button"
+                onAction="#onRehostButtonClicked" >
+            <tooltip><Tooltip text="Rehost"/></tooltip>
+            <graphic>
+                <Region styleClass="icon,host-icon"/>
+            </graphic>
+        </Button>
         <Button fx:id="watchButton" mnemonicParsing="false" styleClass="primary-button"
-                onAction="#onWatchButtonClicked" >
+                onAction="#onWatchButtonClicked">
+            <tooltip><Tooltip text="Watch"/></tooltip>
```

Comment:
Move this to the detailController not the card controller

Reply:
I don't agree with this; it moves the functionality off the front page where its quick to access. Unlike requesting extra detail in the detailController such as chat messages, this is intended to be a 'quick hit' rehost option - equivalent to the delete option on the local replays view.

![image](https://user-images.githubusercontent.com/5471394/156637420-d9b4c1b2-5e0d-4b2b-a03e-2d75c2ce8ef3.png)
&
![image](https://user-images.githubusercontent.com/5471394/156637494-54d2ac55-c759-4e6b-b28f-62bba6938ddc.png)

Reply:
The main concern is just clutter and clarity. I think if we can get a better icon that mean host then it would be okay. Although I am not sure what that icon is at the moment.

Reply:
So something like this:
![image](https://user-images.githubusercontent.com/5471394/156671354-cff33bf8-4cb2-4696-a2da-7a53729f8fd2.png)
I'd change it to be inline with the watch button, but conceptually, around there is ok?

Reply:
Updated in the latest commit.